<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carrega Camadas</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QPushButton" name="load_btn">
     <property name="text">
      <string>&amp;Carregar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="db_label">
      <property name="text">
       <string>Banco de Dados :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="db_options">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Estilo :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="styles_options">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>262</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="layers_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="progress_load">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="load_menu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Adicionar Menu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
